modules/op-default-prunner-configuration.adoc

```diff
@@ -0,0 +1,22 @@
+// This module is included in the following assembly:
+//
+// cicd/pipelines/automatic-pruning-taskrun-pipelinerun.adoc
+
+:_content-type: REFERENCE
+[id="default-prunner-configuration_{context}"]
+= Default prunner configuration
+
+The default configuration for periodic prunning of pipeline runs is as follows:
+
+[source,yaml]
+----
+...
+pruner:
+  keep: 100
+  resources:
+  - pipelinerun
+  schedule: 0 8 * * *
+...
+----
+
+You can override the default prunner configuration by using annotations.
```

Comment:
How about this.
You can override the default prunner configuration for a namespace by using annotations on namespace.